pkg/web/router.go

```diff
@@ -29,6 +30,7 @@ func NewRouter(kratosClient *ik.Client, hydraClient *ih.Client, distFS fs.FS, ba
 		middleware.RequestID,
 		monitoring.NewMiddleware(monitor, logger).ResponseTime(),
 		middlewareCORS([]string{"*"}),
+		middleware.Timeout(5*time.Second),
```

Comment:
IMHO:
1) 5s seems like a short time.
2) Only the deep health check endpoint creates go routines, so if we were to use this I think that we would add it only to that endpoint.
3) from what I understand if you wish to use this middleware then in your handler you need to select the ctx.Done() channel to check for the signal, which is something we don't do.

So I have to ask: why do we want this middleware?